src/module/retry.ts

```diff
@@ -61,23 +61,55 @@ export class Retry extends Module {
   private async _promiseRetry<T> (circuit: Circuit, attempts: number, promise: CircuitFunction, ...params: any[]): Promise<T> {
     if (attempts - 1 === 0) {
       this.emit('retry', circuit, this.attempts);
```

Comment:
Maybe not needed to emit this event if this.attemps is equal to 0 (weird config)

Reply:
Will do!